cakechat/utils/tee_file.py

```diff
@@ -1,11 +1,13 @@
-import cPickle as pickle
 import os
 import tempfile
 
+from six.moves import xrange
+from six.moves import cPickle as pickle
 
 def _pickle_iterable(filename, iterable):
     with open(filename, 'wb') as pickle_fh:
-        pklr = pickle.Pickler(pickle_fh, pickle.HIGHEST_PROTOCOL)
+        # Pickle on HIGHEST_PROTOCOL breaks on Python 3.6.5
+        pklr = pickle.Pickler(pickle_fh, 2)
```

Comment:
It's better to move it to constant like '_PICKLE_HIGHEST_COMPATIBLE_PROTOCOL = 2'